### PR TITLE
add border2

### PR DESCRIPTION
**Pull request by mphongbt**
pleases merge

### CODE REVIEW

File: app/src/main/res/drawable/border2.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="@color/yellow"/>
```
Comment (thong20) [inline app/src/main/res/drawable/border2.xml:3]: i dont like yellow, please change blue

File: app/src/main/res/drawable/border2.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (thong20) [inline app/src/main/res/drawable/border2.xml:2]: please add "stroke" tag color red

Comment (mphongbt) [reply app/src/main/res/drawable/border2.xml:2]: ok baby

**Review by thong20 — COMMENTED** [review]
good job

**Pull request merged**